### ISSUE TITLE
Updated To El Capitan & Mac Pro 5,1 no longer working

### ISSUE THREAD

**Issue by claywd1210** [dokterdok/Continuity-Activation-Tool#242]
Updated my mac pro today after uninstalling cat 2.1. Disabled CSR and enabled kext-dev-mode. Ran cat and received the following. 

"OS X reports Continuity as active.
Print: Entry, "::ContinuitySupport", Does Not Exist
Patching ContinuitySupport...           Set: Entry, "::ContinuitySupport", Does Not Exist
OK.

Press any key to reboot or CTRL-C to cancel...^C
logout
-n Saving session...
-ne 
...copying shared history...
-ne 
...saving history...
-n truncating history files...
-ne 
...
completed.

[Process completed]"

Airdrop window is black and of course none of the continuity features are working. Does anyone know how to approach this one? Already tried uninstalling, reinstalling, signing out and back into iCloud.

**Comment by sysfloat** [comment]
I think I know what causes this, I will probably fix it tomorrow. As a workaround run "System Diagnostics" before attempting to activate Continuity.

**Comment by claywd1210** [comment]
@sysfloat Okay, tried that workaround and here is what I got.
 --- Initiating system compatiblity check ---

--- Hardware/OS checks ---
Verifying Continuity status...          OK. OS X reports Continuity as active
Verifying Mac model reference...        OK. Known compatible Mac Model detected: MacPro5,1
Verifying Mac board-id...               OK. Short board id detected: Mac-F221BEC8
Verifying OS X version...               Warning: This version of Mac OS X (10.11) is Experimental! Only partially tested on El Capitan
Verifying Wi-Fi hardware...             OK. A Broadcom AirPort card is active, and is using the Continuity compatible Brcm4360 kext
Verifying AWDL status...                OK. An AWDL interface is up, Wi-Fi is ready for Continuity
Verifying Bluetooth hardware...         OK. The internal Bluetooth card is active
Verifying Bluetooth version...          OK. Bluetooth 4.0 detected
Verifying Bluetooth features...         OK. Bluetooth features are Continuity compliant
Verifying Bluetooth firmware...         OK. Bluetooth firmware version: v112 c9036
Verifying Login Item...                 OK. Login item for Auto Continuity Check is not set.

--- Modifications check ---
Verifying OS kext protection...         OK. Kext developer mode is not active. This tool can fix this.
Verifying SIP...                        Ok. System Integrity Protection is already disabled
Verifying ContinuitySupport...          OK. This tool can fix this.
Verifying kexts readability...          OK. Wi-Fi and Bluetooth kexts were found and could be read
Verifying Wi-Fi whitelist status...     OK. Your board-id is not yet whitelisted. This tool can fix this.
Verifying BT4 dongles compatibility...  OK. Compatibility with BT4 USB dongles is not enabled, this tool can fix this if a dongle is plugged in
Verifying old Wi-Fi kext presence...    OK. Legacy Brcm4331 Wi-Fi driver is present. This tool can fix this.
Verifying legacy Wi-Fi card patch...    OK. The legacy Wi-Fi patch is not present. This tool can fix this.
--- Modifications check ---

Press any key to go back to the main menu...

OS X reports Continuity as active.
Patching ContinuitySupport...           OK.

Press any key to reboot or CTRL-C to cancel...

Then if i run the diagnostic again it reports the same thing.

**Comment by sysfloat** [comment]
The new version (2.2.1) should fix this issue.

**Comment by claywd1210** [comment]
I wish that were the case but I still have no airdrop and no proper handoff, instant hotspot and now messages will not sign in. 

Tried signing in and out of iCloud but I found no fix.

**Comment by claywd1210** [comment]
So after doing a fresh install and trying again here is what I got. 

I choose install and I get 
----------------------------Begin Terminal Output-------------------------
Press any key to go back to the main menu...

OS X reports Continuity as active.
Patching ContinuitySupport... OK.

Press any key to reboot or CTRL-C to cancel...

----------------------------End Terminal Output-------------------------

Then I restart the computer and run the diagnostic
----------------------------Begin Terminal Output-------------------------
--- Initiating system compatiblity check ---

--- Hardware/OS checks ---
Verifying Continuity status... OK. OS X reports Continuity as active
Verifying Mac model reference... OK. Known compatible Mac Model detected: MacPro5,1
Verifying Mac board-id... OK. Short board id detected: Mac-F221BEC8
Verifying OS X version... Warning: This version of Mac OS X (10.11) is Experimental! Only partially tested on El Capitan
Verifying Wi-Fi hardware... OK. A Broadcom AirPort card is active, and is using the Continuity compatible Brcm4360 kext
Verifying AWDL status... OK. An AWDL interface is up, Wi-Fi is ready for Continuity
Verifying Bluetooth hardware... OK. The internal Bluetooth card is active
Verifying Bluetooth version... OK. Bluetooth 4.0 detected
Verifying Bluetooth features... OK. Bluetooth features are Continuity compliant
Verifying Bluetooth firmware... OK. Bluetooth firmware version: v112 c9036
Verifying Login Item... OK. Login item for Auto Continuity Check is not set.

--- Modifications check ---
Verifying OS kext protection... OK. Kext developer mode is not active. This tool can fix this.
Verifying SIP... Ok. System Integrity Protection is already disabled
Verifying ContinuitySupport... OK. This tool can fix this.
Verifying kexts readability... OK. Wi-Fi and Bluetooth kexts were found and could be read
Verifying Wi-Fi whitelist status... OK. Your board-id is not yet whitelisted. This tool can fix this.
Verifying BT4 dongles compatibility... OK. Compatibility with BT4 USB dongles is not enabled, this tool can fix this if a dongle is plugged in
Verifying old Wi-Fi kext presence... OK. Legacy Brcm4331 Wi-Fi driver is present. This tool can fix this.
Verifying legacy Wi-Fi card patch... OK. The legacy Wi-Fi patch is not present. This tool can fix this.
--- Modifications check ---

----------------------------Begin Terminal Output-------------------------


Not sure why this keeps happening but I can still no longer sign into iMessages. It keeps telling me there was an error activating device. 

No iMessages, Instant Hotspot, Handoff,  or working airdrop.

**Comment by Claywd** [comment]
So this latest attempt definitely breaks down somewhere. I modified the script such that it would run all the task and patch the system. I also removed kext-dev-mode. So I can confirm the following. 

1) handled, instant hotshot, &  messages (sms relay and simply in logging in) are not working
2) air drop works beautifully. IOS and OS X alike
3) the tool looks as though it may be using old and outdated files

**Comment by sysfloat** [comment]
I guess @Claywd and @claywd1210 are the same person?
Anyways, send me an email at catbetaelcap@gmail.com and I will try to fix this with you. It seems like your board-id does not get patched correctly.

**Comment by claywd1210** [comment]
Yeah they are. I figured this out eventually. It went all the way back to issues with the open directory service on my Mac Pro. Handoff and Open Directory don't really like one another from my experience. Most of the issues comes from the device authorization trying to authorize through the directory server and messages federation instead of apple servers. 

Once I reinstalled the system, applied patches and hacks and then used continuity activator, it worked like a charm.
